packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -3,531 +3,823 @@ module.exports = {
     // enforce line breaks after opening and before closing array brackets
     // https://eslint.org/docs/rules/array-bracket-newline
     // TODO: enable? semver-major
-    'array-bracket-newline': ['off', 'consistent'], // object option alternative: { multiline: true, minItems: 3 }
+    // KEEP AS IS
+    "array-bracket-newline": ["off", "consistent"], // object option alternative: { multiline: true, minItems: 3 }
 
     // enforce line breaks between array elements
     // https://eslint.org/docs/rules/array-element-newline
     // TODO: enable? semver-major
-    'array-element-newline': ['off', { multiline: true, minItems: 3 }],
+    // KEEP, SITUATIONAL ANALYSIS NEEDED
+    // Need to determine how many items to allow on single line before breaking out to 1 per line
```

Comment:
Evaluate with Prettier ESLint recommended rules. I.e. it turns off ESLint rules that Prettier will impact.

Reply:
agree, also need to determine the minimum number of items required before breaking lines.